src/sentry/backup/services/import_export/model.py

```diff
@@ -27,7 +27,7 @@ class RpcFilter(RpcModel):
     Shadows `sentry.backup.helpers.Filter` for the purpose of passing it over an RPC boundary.
     """
 
-    model_name: str
+    import_model_name: str
```

Comment:
Let's change this to `for_model`. `Filter`/`RpcFilter` is used both when importing and exporting, so calling this `import_...` will be confusing in the latter case. I think omitting the `name` suffix is okay in this case, since said in English it would be something like "this is a `Filter` *on model* X`"

Reply:
Whoops, meant `on_model` in that first sentence!